### PR TITLE
test: add test cases for sparse vector

### PR DESCRIPTION
**Pull request by elstic**
issue: https://github.com/milvus-io/milvus/issues/31483
pr:  [#32155](https://github.com/milvus-io/milvus/pull/32155)

### CODE REVIEW

File: tests/python_client/testcases/test_search.py [hunk 1]
```diff
@@ -11947,3 +11947,91 @@ def test_hybrid_search_result_order(self):
...
+        collection_w.load()
+        collection_w.search(data[-1][-1:], ct.default_sparse_vec_field_name,
+                            ct.default_sparse_search_params, default_limit,
+                            check_task=CheckTasks.check_search_results,
+                            check_items={"nq": default_nq,
+                                         "limit": default_limit})
+
+    @pytest.mark.tags(CaseLabel.L2)
+    @pytest.mark.parametrize("index, index_params", zip(ct.all_index_types[9:11], ct.default_index_params[9:11]))
+    @pytest.mark.parametrize("dim", [1, 16, 512, 32768, 524288])
```
Comment (yanliang567) [inline tests/python_client/testcases/test_search.py:11981]: i think it is enough to cover dim min and max

Comment (elstic) [reply tests/python_client/testcases/test_search.py:11981]: > i think it is enough to cover dim min and max

The maximum vector dimension of sparse vector is uint32max-1, which is too large, so it is not covered. 
uint32max - 1 = 4294967294

Comment (elstic) [reply tests/python_client/testcases/test_search.py]: Changed to max and min dim validation, exception cases smaller than min dim and larger than max dim added later in pr

**Pull request closed without merging**